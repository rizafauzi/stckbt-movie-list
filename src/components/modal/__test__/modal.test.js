

import React from 'react'
import ReactDOM from 'react-dom'
import Modal from '../modal.component'
import { cleanup } from '@testing-library/react'
import '@testing-library/jest-dom'

import renderer from 'react-test-renderer'

afterEach(cleanup)

it("renders without crashing", () => {
  const div = document.createElement('div')
  ReactDOM.render(<Modal />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it('it matches snapshot', () => {
  const tree = renderer.create(<Modal />).toJSON()
  expect(tree).toMatchSnapshot()
})