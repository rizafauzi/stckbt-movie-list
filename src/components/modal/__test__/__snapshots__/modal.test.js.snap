// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<div
  className="sc-bdnylx lgJeVP"
>
  <div
    className="sc-gtssRu bMnysw"
  />
  <div
    className="sc-dlnjPT ieGyKG"
  />
</div>
`;
